installer-clusters.cfg

```diff
@@ -1 +1 @@
-mainnet mainnet_flight testnet shelley_qa staging
```

Comment:
@tomothespian we must not forget to revert this change prior to merging this PR 🙏